pkg/config/config.go

```diff
@@ -12,13 +12,17 @@ type config struct {
 	DBConnectionString string
 	DBMaxConnections   int
 	ListenAddress      string
+	LogLevel           string
+	LogFormat          string
 }
 
 func GetConfig() config {
 	return config{
 		DBConnectionString: getDBConnectionString(),
 		DBMaxConnections:   getEnvInt("DBMAX_CONNECTIONS", 100),
 		ListenAddress:      fmt.Sprintf(":%v", getEnvInt("PORT", 3000)),
+		LogLevel:           strings.ToLower(getEnv("LOG_LEVEL", "info")),
+		LogFormat:          strings.ToLower(getEnv("LOG_FORMAT", "text")), //can be text or json
```

Comment:
Should json be the default?
I'm thinking it would allow easier integration with centralised logging in openshift where you can qeury logs of containers based on json fields

Thoughts @maleck13 @darahayes

Reply:
I think this is fine. The only (minor) concern I would have is if writing lots of json logs might impact performance, this seems like a performance sensitive application

Reply:
I'm good with it staying as text then.
We can always change the default, or the admin can change it if needed.

Reply:
+1 on the performance point. Serializing JSON introduces bottlenecks. At a later stage we can gather some actual perf data and optimize accordingly.